composeApp/src/commonMain/kotlin/world/respect/app/view/apps/list/AppListScreen.kt

```diff
@@ -0,0 +1,117 @@
+package world.respect.app.view.apps.list
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Link
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import org.jetbrains.compose.resources.stringResource
+import respect.composeapp.generated.resources.Res
+import respect.composeapp.generated.resources.add_from_link
+import respect.composeapp.generated.resources.add_link
+import world.respect.app.app.RespectAsyncImage
+import world.respect.app.appstate.getTitle
+import world.respect.app.viewmodel.apps.list.AppListUiState
+import world.respect.app.viewmodel.apps.list.AppListViewModel
+import world.respect.datasource.DataLoadResult
+import world.respect.datasource.DataLoadState
+import world.respect.datasource.compatibleapps.model.RespectAppManifest
+
+@Composable
+fun AppListScreen(
+    viewModel: AppListViewModel
+) {
+    val uiState by viewModel.uiState.collectAsState()
+
+    AppListScreen(
+        uiState = uiState,
+        onClickAddLink = { viewModel.onClickAddLink() },
+        onClickApp = { viewModel.onClickApp(it) }
+    )
+}
+
+@Composable
+fun AppListScreen(
+    uiState: AppListUiState,
+    onClickAddLink: () -> Unit,
+    onClickApp: (DataLoadState<RespectAppManifest>) -> Unit
+) {
+    val key=stringResource(Res.string.add_link)
+
+    LazyColumn(
+        modifier = Modifier.fillMaxSize(),
+    ) {
+        item(key=key) {
```

Comment:
There is no need to get a string resource here - the key is never shown to the user. We can use any string or other sensible constant.